fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/SkolemNodeRdfContext.java

```diff
@@ -93,6 +94,10 @@ public SkolemNodeRdfContext(final FedoraResource resource,
             return null;
 
         } catch (final RepositoryException e) {
+            if (e instanceof ItemNotFoundException) {
+                // Effectively skip resources that do not exist
```

Comment:
Either your PR works, in which case there is no need to skip non-existent resources, or your PR does not work.